dev_install.adoc

```diff
@@ -16,6 +16,15 @@ Configure authentication for the cluster using one of the following options:
 Option #1: Contact a member of the Dev Sandbox Team for instructions on how to configure the cluster to use our internal Dev SSO. +
 Option #2: Configure your own Keycloak server and set up authentication on the OpenShift cluster: https://docs.openshift.com/container-platform/4.6/authentication/configuring-internal-oauth.html
 
+
+Option #3: Deploy and configure keycloak internally as part of the cluster. Just add `DEV_SSO=true` parameter to the `dev` targets. For eg.: `make dev-deploy-latest DEV_SSO=true` will deploy latest version of the operators with a preconfigured keycloak instance.
+If you are presented with the following error, you just need to ignore the certificate problem for the `https://keycloak-<dev-sso-namespace>.<domain>/auth/js/keycloak.js :
```

Comment:
you can go to the shorter version of the URL which is also printed at the end of the command.
```suggestion
If you are presented with the following error, then you need to accept the self-signed certificate of the dev Keycloak instance first. Go to `https://keycloak-<dev-sso-namespace>.<domain>/auth (the complete link is printed out at the end of the command) and accept the certificate.:
```